test/chpldoc/globals/lifetimeChecking.doc.good

```diff
@@ -23,7 +23,7 @@ or
 
 .. data:: var b = new unmanaged Foo()
 
-.. data:: var c = new borrowed Foo()
+.. data:: var c = new owned Foo().borrow()
```

Comment:
Should the documentation for these cases have the parens around `new owned Foo()` that are present in the source?

Reply:
Probably. @lydia-duncan do you think it should? The source is `var c = (new owned Foo()).borrow()`.

Reply:
We are in the process of replacing chpldoc with dyno-chpldoc so I suggest we not worry about this for now.

Reply:
I think it would make things clearer but I suspect the parentheses have not been preserved.  But I agree with Michael,  think it's fine to punt that to dyno-chpldoc